src/utils/SafeTransferLib.sol

```diff
@@ -34,105 +33,104 @@ library SafeTransferLib {
         address to,
         uint256 amount
     ) internal {
-        bool callStatus;
+        bool success;
 
         assembly {
             // Get a pointer to some free memory.
             let freeMemoryPointer := mload(0x40)
 
-            // Write the abi-encoded calldata to memory piece by piece:
-            mstore(freeMemoryPointer, 0x23b872dd00000000000000000000000000000000000000000000000000000000) // Begin with the function selector.
-            mstore(add(freeMemoryPointer, 4), and(from, 0xffffffffffffffffffffffffffffffffffffffff)) // Mask and append the "from" argument.
-            mstore(add(freeMemoryPointer, 36), and(to, 0xffffffffffffffffffffffffffffffffffffffff)) // Mask and append the "to" argument.
-            mstore(add(freeMemoryPointer, 68), amount) // Finally append the "amount" argument. No mask as it's a full 32 byte value.
+            // Write the abi-encoded calldata into memory, beginning with the function selector.
+            mstore(freeMemoryPointer, 0x23b872dd00000000000000000000000000000000000000000000000000000000)
+            mstore(add(freeMemoryPointer, 4), from) // Mask and append the "from" argument.
+            mstore(add(freeMemoryPointer, 36), to) // Mask and append the "to" argument.
```

Comment:
comments say mask and append but didn't you remove the mask?

Reply:
oop